crates/broker/src/order_monitor.rs

```diff
@@ -490,18 +481,8 @@ where
         });
 
         tracing::debug!(
-            "Orders ready for proving, prioritized. Before applying capacity limits: {:?}",
-            orders
-                .iter()
-                .map(|order| format!(
-                    "{} [Lock expires at: {} ({} seconds from now), Expires at: {} ({} seconds from now)]",
-                    order.id(),
-                    order.request.lock_expires_at(),
-                    order.request.lock_expires_at().saturating_sub(now_timestamp()),
-                    order.request.expires_at(),
-                    order.request.expires_at().saturating_sub(now_timestamp())
-                ))
-                .collect::<Vec<_>>()
+            "Orders ready for proving, prioritized. Before applying capacity limits: {}",
+            orders.iter().map(|order| format!("{order}")).collect::<Vec<_>>().join(", ")
```

Comment:
```suggestion
            orders.iter().map(ToString::to_string).collect::<Vec<_>>().join(", ")
```

Optional: could do something like this I think